### DISCUSSION TITLE
Is there a place where I can signup to the new GitHub issue features?

### DISCUSSION THREAD

**Discussion by xunafay** [community/events#47] [Cloud]
I love the look of the new tasklist! Is there a way to sign up to this beta or is this closed for now?

**Comment by queenofcorgis** [comment]
Find the waitlist [here](https://github.com/features/issues/signup).

**Comment by orvn** [comment]
Does this also apply to the iOS Testflight which was just mentioned for beta Github Projects features?

**Comment by oleg-nenashev** [comment]
@xunafay We got an individual an early alpha invite in Keptn, so you can check out a few examples and try out the UX in Keptn repositories, e.g. https://github.com/keptn/keptn.github.io/issues/1484 . No public GitHub Issue Management sandbox there though
